File: frontend/src/Pages/Auth/VerificationPage.js
import React, {useEffect} from "react";
import VerificationForm from "../../Components/Forms/VerificationForm";
import LoginNavbar from "../../Components/Static/LoginNavbar";

const VerificationPage = () => {
    useEffect(() => {
        document.title = "Verification";
    }, []);
    return (
        <div>
            <LoginNavbar />
            <h1>Verification</h1>
            <VerificationForm />
        </div>
    );
}

export default VerificationPage;

File: flutter_frontend/lib/pages/auth/verificationPage.dart
import 'package:flutter/material.dart';
import 'package:flutter_frontend/assets/auth/forms/verificationForm.dart';

class VerificationPage extends StatelessWidget {
  const VerificationPage({super.key});

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(title: const Text('Verification')),
      body: Center(
        child: Padding(
          padding: const EdgeInsets.all(16.0),
          child: VerificationForm(),
        ),
      ),
    );
  }
}
